tavla/app/(admin)/edit/[id]/components/MetaSettings/actions.ts

```diff
@@ -50,15 +70,20 @@ export async function saveLocation(bid: TBoardID, location?: TLocation) {
 
     const board = await getBoard(bid)
     if (!board) return getFormFeedbackForError('board/not-found')
-    await firestore()
-        .collection('boards')
-        .doc(bid)
-        .update({
-            tiles: await getTilesWithDistance(board, location),
-            'meta.location': location ?? firestore.FieldValue.delete(),
-            'meta.dateModified': Date.now(),
-        })
-    revalidatePath(`/edit/${bid}`)
+
+    try {
+        await firestore()
+            .collection('boards')
+            .doc(bid)
+            .update({
+                tiles: await getTilesWithDistance(board, location),
+                'meta.location': location ?? firestore.FieldValue.delete(),
```

Comment:
enda en rar ting (som heller ikke egentlig er relatert til denne PR-en) er at vi er veldig inconsistent med hvilke felter som slettes hvis de er tomme 🤔 hvorfor sletter vi location når den fjernes, men ikke tags, walking-distance og whitelistedLines?

Reply:
..men dette er mer en generell kommentar enn noe som skal fikses i denne PR-en lol

Reply:
Enig! Jeg lager kort for denne og den over